lib/valkyrie/persistence/fedora/query_service.rb

```diff
@@ -17,6 +22,9 @@ def find_by(id:)
     end
 
     # (see Valkyrie::Persistence::Memory::QueryService#find_by_alternate_identifier)
+    # Find a resource using its Valkyrie::ID
```

Comment:
I think you can remove all the documentation here - the see blabla bit imports the documentation from the memory adapter. Ditto for all these methods.